cubic-server/Player.cpp

```diff
@@ -159,7 +165,14 @@ void Player::setPosition(const Vector3<double> &pos)
 {
     auto newChunkPos = Position2D(transformBlockPosToChunkPos(pos.x), transformBlockPosToChunkPos(pos.z));
     auto oldChunkPos = Position2D(transformBlockPosToChunkPos(_pos.x), transformBlockPosToChunkPos(_pos.z));
-
+    auto _pos2d = Vector2<double>(_pos.x, _pos.z);
+    auto pos2d = Vector2<double>(pos.x, pos.z);
```

Comment:
Please name those differently, we can't know here what variable represents what.